### PR TITLE
Python: small fixes

### PR DESCRIPTION
**Pull request by kwek20**
# Description of change

Ran node info, works now.

Also did an fmt on rust, how did that pass :D

### CODE REVIEW

File: bindings/python/iota_sdk/client/client.py [hunk 1]
```diff
@@ -321,6 +322,13 @@ def get_node(self) -> Dict[str, Any]:
         """
         return self._call_method('getNode')
 
+    def get_info(self) -> NodeInfoWrapper:
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/client/client.py]: We already have this method in `bindings/python/iota_sdk/client/_node_core_api.py` ?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/client/client.py]: Yes and even without `print(f'{result}')` :trollface:

Comment (kwek20) [reply bindings/python/iota_sdk/client/client.py]: i swear, i looked everywhere and used the search :D it didnt show!!

File: bindings/python/iota_sdk/client/client.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 from iota_sdk.types.block.signed_block import UnsignedBlock
 from iota_sdk.types.common import HexStr, Node
 from iota_sdk.types.feature import Feature
+from iota_sdk.types.node_info import NodeInfoWrapper
```
Comment (Thoralf-M) [inline bindings/python/iota_sdk/client/client.py]: Remove as well?

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**